source/commands/danger.ts

```diff
@@ -32,9 +32,13 @@ program
     console.log("       http://danger.systems/js/reference.html")
   })
 
+// Commander mutates process.argv
+const originalProcessArgV = process.argv
```

Comment:
Don't we need to actually copy the array here, not just grab a reference to it?

Like `[...process.argv]` or something. Otherwise we've just got a second reference to the mutated array, I'd think.